optuna/integration/allennlp.py

```diff
@@ -88,7 +84,7 @@ def __init__(
         include_package: Optional[Union[str, List[str]]] = None
     ):
 
-        _check_allennlp_availability()
+        _imports.check()
```

Comment:
The error message is shown here is as follows:

```console
>>> optuna.integration.AllenNLPExecutor(None, None, None)
/tmp/optuna/optuna/_experimental.py:83: ExperimentalWarning: AllenNLPExecutor is experimental (supported from v1.4.0). The interface can change in the future.
  ExperimentalWarning,
Traceback (most recent call last):
  File "/tmp/optuna/optuna/integration/allennlp.py", line 16, in <module>
    import _jsonnet
ModuleNotFoundError: No module named '_jsonnet'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/optuna/optuna/_experimental.py", line 86, in wrapped_init
    _original_init(self, *args, **kwargs)
  File "/tmp/optuna/optuna/integration/allennlp.py", line 87, in __init__
    _imports.check()
  File "/tmp/optuna/optuna/_imports.py", line 80, in check
    raise exc_type(self._message) from exc_value
ModuleNotFoundError: Failed to import an optional package. See causing exceptions for details.
```

The import error is about `_jsonnet`, which is one of the dependencies of AllenNLP.
How about specifying the module name with the argument of the `try_import` function?